client/app/services/query.js

```diff
@@ -103,6 +136,39 @@ class Parameter {
   set ngModel(value) {
     this.setValue(value);
   }
+
+  toQueryParams() {
```

Comment:
This is a bit of a confusing name, maybe `toUrlQueryStringParams`? (same with the `from` variant of the function).